suds/wsse.py

```diff
@@ -139,7 +140,7 @@ def setnonce(self, text=None):
             s.append(self.username)
             s.append(self.password)
             s.append(Token.sysdate())
-            m = md5()
+            m = sha56()
```

Comment:
I think this call should be for **sha256**, the above code would not work for python v3.7.